ADOPTERS.md

```diff
@@ -12,6 +12,7 @@
 - [In Loco](https://inloco.com.br/en/)
 - [LeadIQ](https://leadiq.com)
 - [Facelift](https://www.facelift-bbt.com/en)
+- [Mulligan Funding] (https://www.mulliganfunding.com/)
```

Comment:
Looks like the Markdown for this isn't rendering properly --- I think it needs to be:
```suggestion
- [Mulligan Funding](https://www.mulliganfunding.com/)
```